common/flatpak-dir.c

```diff
@@ -6946,14 +6946,57 @@ flatpak_dir_undeploy_all (FlatpakDir   *self,
   return TRUE;
 }
 
+/**
+ * flatpak_dir_remove_ref:
+ *
+ * Remove the flatpak ref given by @remote_name:@ref from the underlying
+ * OSTree repo. Attempting to remove a ref that is currently deployed
+ * is an error, you need to uninstall the flatpak first. Note that this does
+ * not remove the objects bound to @ref from the disk, you will need to
+ * call flatpak_dir_prune to do that.
```

Comment:
Nitpick: add `()` after `flatpak_dir_prune` (no space between them) and gtk-doc will automatically add a cross-reference link to the function.